discussion_agents/cog/functional/react.py

```diff
@@ -20,19 +23,27 @@ def _build_agent_prompt(question: str, scratchpad: str) -> str:
     Args:
         question (str): The question to be included in the prompt.
         scratchpad (str): Additional scratchpad information to be included.
+        benchmark_type (str): FEVER / HotpotQA
 
     Returns:
         str: A formatted prompt template ready for use.
     """
+    
+    if benchmark_type == "FEVER":
+        examples = REACT_WEBTHINK_SIMPLE3_FEVER_EXAMPLES
+    else:
+        examples = REACT_WEBTHINK_SIMPLE6_FEWSHOT_EXAMPLES
```

Comment:
I'd suggest removing benchmark_type and just adding another variable for "examples"